src/plays/react-todo-app/react-todo.css

```diff
@@ -0,0 +1,152 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.play-details-body {
+    background-image: linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%);
+    color: white;
+    font-family: "Poppins", sans-serif;
+    min-height: 100vh;
+}
+
+.top {
+    font-size: 2rem;
+}
+
+.top,
+form {
```

Comment:
don't use the HTML selectors directly in CSS as they may conflict with an existing style. Please scope them with a class name related to your play.